### ISSUE TITLE
Pop & Plush should have their own space

### ISSUE THREAD

**Issue by stanislas-m** [gobuffalo/gobuffalo#88]
Pop and Plush are big components in Buffalo ecosystem, and they have their own pace. They should have their own namespace or subdomain and a dedicated documentation.

Buffalo documentation should refer to these docs, and only explain how to use Plush and Pop in Buffalo context.

**Comment by markbates** [comment]
Agreed! Tell me what you need and I’ll set up dns entries and heroku. Or we can keep it the same app, and switch template folders based on the domain.

**Comment by koctya** [comment]
Do you have a place for these. I can help out with Pop docs while I'm learning Buffalo.

**Comment by stanislas-m** [comment]
@koctya I've not decided yet, but you can help on the current docs for Pop. :)
